### PR TITLE
CONSOLE-2371: [i18n] Externalize strings in ./packages/operator-lifecycle-manager/src

### PR DESCRIPTION
**Pull request by TheRealJon**
Externalize strings in files at root level of `frontend/packages/operator-lifecycle-manager/src`

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts:17]: We don't need to localize strings that are only printed to the JS console. (It might be better not to in fact since these are meant for developers to debug problems.)

File: frontend/packages/operator-lifecycle-manager/src/utils.ts
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/utils.ts:19]: No need to localize this string.

File: frontend/packages/operator-lifecycle-manager/src/status/csv-status.ts
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/status/csv-status.ts:42]: I think it's less error-prone to do it this way as long as we always use this function to show the status. Then you don't risk missing a string or having the i18next-parser comments get out of sync with the enum value.

```suggestion
        status = i18n.t('olm~OK');
```

**Review by rhamilto — APPROVED** [review]
/lgtm

/hold for approvals

/assign @yapei @ahardin-rh @sferich888

File: frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts
Comment (yapei) [inline frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts:30]: `Operator Description` heading is on the same line with `csvDescription`, please see below screenshot, this needs to be updated
Screen Shot 2020-12-15 at 3 23 53 PM

**Comment by rhamilto** [comment]
@TheRealJon, it looks like you need to run `yarn i18n` and commit the results.

**Comment by TheRealJon** [comment]
@yapei I've addressed your comments if you could review again.

**Comment by yapei** [comment]
@TheRealJon  I've tested again and no more issues

File: frontend/packages/operator-lifecycle-manager/src/utils/useClusterServiceVersions.tsx
Comment (TheRealJon) [inline frontend/packages/operator-lifecycle-manager/src/utils/useClusterServiceVersions.tsx:45]: @yapei This line fixes your comment [here](https://github.com/openshift/console/pull/7496/files?file-filters%5B%5D=.js&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r543106521)

**Comment by rhamilto** [comment]
@TheRealJon rebase required.

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
/hold cancel

@TheRealJon FYI, needs rebase

**Comment by TheRealJon** [comment]
@spadgett rebased. Needs lgtm again.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**
